### DISCUSSION TITLE
NScripter2 ディスカッション(質問・機能追加要望・バグ報告等)

### DISCUSSION THREAD

**Discussion by NaokiTakahashi** [NaokiTakahashi/NScripter2#1] [Announcements]
NScripter2についての掲示板です。質問やバグ報告等に使ってください。

**Comment by Cinseiqa** [comment]
こんにちは、不具合報告です。
NScripter2、2026年4月26日時点での最新版にて、
ノベル（全画面）モードでデフォルトのバックログを表示してからゲーム画面に戻ると、すでに表示された表示文に新しい文章が重なって表示されます。
憶測ですが、バックログの表示時のspformatの表示位置をそのまま取得して、表示文に反映していると思われます。（違っていたらすみません）

なお、サンプルスクリプトでも以下と同様の手順で再現できました。

１．表示文を描写（表示）
２．バックログ（サンプルのもの）を右クリックなどで開く
３．ゲーム画面に戻る
４．そのままクリック待ちを進めると、表示文が特定の位置から再開して新しい表示文が重なる

ＡＤＶモードでは問題ありませんでしたので、付け加えておきます。
